### PR TITLE
Fix instructor search page ui test

### LINKED ISSUE #3765: Fix instructorSearch JSTL
Part of #3762 
@taniach use this for your PR

### PR DESCRIPTION
**Pull request by taniach**
Fixes #3765 
Changes to HTML files

Instructor search page UI test passes now

### CODE REVIEW

File: src/test/resources/pages/InstructorSearchPageSearchCommentsAsHelper.html [hunk 1]
```diff
@@ -81,7 +80,17 @@
...
+                       <div class="col-md-2">
+                          <strong>
+                             Session: searchUI.First feedback session (searchUI.idOfTypicalCourse1)
+                          </strong>
+                       </div>
+                       <div class="col-md-10">
+                       </div>
+                    </div>
+         </div>
                 </div>
```
Comment (wkurniawan07) [inline src/test/resources/pages/InstructorSearchPageSearchCommentsAsHelper.html]: is this relevant?

**Comment by wkurniawan07** [comment]
See if you can make it work without having to add `title=""`. The way it is now is okay, but would be better if you can minimize the changes.

File: src/test/resources/pages/InstructorSearchPageSearchCommentsAsHelper.html (hunk 1 above)
Comment (taniach) [inline src/test/resources/pages/InstructorSearchPageSearchCommentsAsHelper.html]: It is because of a change in the logic .

**Comment by taniach** [comment]
The title="" appears because I use the ElementTag class for the links. The attributes are stored in ElementTag and retrieved with foreach. Hence it is a difficult to exclude only the title.

**Comment by wkurniawan07** [comment]
Go ahead and merge first, I'll settle the rest afterwards.

**Pull request merged**